doc/lsst.meas.extensions.scarlet/overview.rst

```diff
@@ -0,0 +1,217 @@
+The main differences are the underlying architecture and the python modules used to optimize the solution due to differences in intended application.
+
+*scarlet* was designed to be an all-purpose model-fitting deblender for a wide range of instruments, including simultaneous processing.
+It can create a joint model from observations of ground based photometry, space based photometry, and even spectral cameras like grisms.
+This requires a robust and complicated class hierarchy in order to properly transform data into the proper spectral and spatial pixels of the underlying model, including using a separate package (`autograd <https://github.com/HIPS/autograd>`_) to create gradients used during optimization (see `optimization`_).
+
+*scarlet-lite* was designed specifically for use in the science pipelines and is therefor able to run in a smaller memory footprint in a fraction of the run time of scarlet main.
+In this case the data is assumed to be organized as if it was taken from a hyper-spectral camera, meaning the observation in each band is covering the same spatial area, in the same orientation, on the same pixel grid.
+This allows for a much simpler class hierarchy and gradients that can be calculated analytically, avoiding the use of an external automatic differentiation package and the memory and runtime overhead that creates.
+
+.. note::
+    The *scarlet* developers are currently working on porting from ``autograd`` to either `JAX <https://github.com/google/jax>`_ or `pytorch <https://github.com/pytorch/pytorch>`_.
+    In either case, the general performance issues present in ``autograd`` will still be present.
+
+However, the underlying models and algorithms in scarlet and scarlet-lite are *identical*, the only difference between them is the architecture of the code used to generate and optimize the models.
+For this reason the remainder of this document will refer to the multi-band deblending algorithm as *scarlet*, even though in practice the actual package implementing the algorithm is *scarlet lite*.
+
+.. _basic_scarlet_model:
+
+Basic Model
+^^^^^^^^^^^
+
+Just like the single-band deblender, in order to make a model for a multi-band image we will have to make some assumptions.
+In this case the main assumption is that an astrophysical source can be modeled as a collection of components, where each component can be described by a single SED that is roughly constant but can vary in magnitude over its spatial extent (morphology).
+This assumption can be justified physically by considering the structure of a galaxy.
+In general the populations of stars in the bulge and disk are different, as well as perhaps additional populations of star forming regions, each with its own distinct SED.
+However, the actual colors in each population are not that different, which is why parametric bulge-disk models (Sersic models, Spergel models, Gaussian mixture models, etc.) have been successfully used in deblending and photometric measurement.
```

Comment:
Example: star forming regions in one part of the galaxies are spectrally similar to star forming regions in other parts of the galaxy since they are made up of the same population of stars.

Reply:
Ok, it sounded ambiguous so I suggest changing it to "the range of colors within each population is not that large", and potentially add something like "e.g. star-forming regions are blue, and bulges tend to be red".